inst/pages/miaverse.qmd

```diff
@@ -120,6 +120,7 @@ The following DA methods support `(Tree)SummarizedExperiment`.
 - [ANCOMBC](https://bioconductor.org/packages/devel/bioc/html/ANCOMBC.html) [@ancombc2020] for differential abundance analysis
 - [benchdamic](https://bioconductor.org/packages/release/bioc/vignettes/benchdamic/inst/doc/intro.html) [@Calgaro2022] for benchmarking differential abundance methods
 - [ALDEx2](https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html) [@Gloor2016] for differential abundance analysis
+- [lefser](https://www.bioconductor.org/packages/release/bioc/html/lefser.html) [@Asya2024] for metagenomic biomarker discovery
```

Comment:
This could be sorted into alphabetical order

Reply:
Yes - although LEfSe claims to be biomarker discovery tool and that's different from DAA (perhaps there is overlap?). Could be also grouped thematically.